src/connectors/kessel/impl.js

```diff
@@ -0,0 +1,205 @@
+'use strict';
+
+const assert = require('assert');
+const Connector = require('../Connector');
+const log = require('../../util/log');
+
+const metrics = require('../metrics');
+
+// Import new Kessel SDK with ClientBuilder
+let ClientBuilder, fetchOIDCDiscovery, OAuth2ClientCredentials, OAuth2AuthRequest, fetchDefaultWorkspace
+try {
+    // Import the new ClientBuilder from the updated SDK
+    const kesselSdk = require('@project-kessel/kessel-sdk/kessel/inventory/v1beta2');
+    ClientBuilder = kesselSdk.ClientBuilder;
+
+    const oAuthSdk = require('@project-kessel/kessel-sdk/kessel/auth');
```

Comment:
Lets import the Allowed enum, will be used below

```suggestion
let ClientBuilder, fetchOIDCDiscovery, OAuth2ClientCredentials, OAuth2AuthRequest, fetchDefaultWorkspace, Allowed
try {
    // Import the new ClientBuilder from the updated SDK
    const kesselSdk = require('@project-kessel/kessel-sdk/kessel/inventory/v1beta2');
    ClientBuilder = kesselSdk.ClientBuilder;
    
    const allowedSdk = require('@project-kessel/kessel-sdk/kessel/inventory/v1beta2/allowed');
    Allowed = allowedSdk.Allowed;

    const oAuthSdk = require('@project-kessel/kessel-sdk/kessel/auth');
```